components/normalizer/src/lib.rs

```diff
@@ -180,12 +196,12 @@ fn decomposition_starts_with_non_starter(trie_value: u32) -> bool {
 ///
 /// The trie value must not be one that signifies a special non-starter
 /// decomposition. (Debug-only)
-fn ccc_from_trie_value(trie_value: u32) -> CanonicalCombiningClass {
+fn ccc_from_trie_value(trie_value: u32) -> u8 {
```

Comment:
nit: define `type CanonicalCombiningClass = u8` or a newtype, and then qualify the properties one?

Reply:
This is a good idea, I did one better, I wrote a seamless shim type `CanonicalCombiningClass(pub u8)` that gets imported in non-properties mode.